tests/security/stig/oscap_xccdf_eval.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Test "stig" hardening in the "scap-security-guide": detection mode
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#93886, poo#104943
+
+use base 'stigtest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    select_console 'root-console';
+
+    # Get ds file and profile ID
+    my $f_ssg_sle_ds = $stigtest::f_ssg_sle_ds;
+    my $profile_ID = $stigtest::profile_ID;
+    my $f_stdout = $stigtest::f_stdout;
+    my $f_stderr = $stigtest::f_stderr;
+    my $f_report = $stigtest::f_report;
+
+    # Verify detection mode
+    my $ret = script_run("oscap xccdf eval --profile $profile_ID --oval-results --report $f_report $f_ssg_sle_ds > $f_stdout 2> $f_stderr", timeout => 300);
+    if ($ret == 0) {
+        record_info('PASS');
+    } elsif ($ret == 1 || $ret == 2) {
+        record_info("errno=$ret", "# oscap xccdf eval --profile $profile_ID\" returns: $ret");
```

Comment:
just curious about it shows in openQA 
`# oscap xccdf eval --profile xccdf_org.ssgproject.content_profile_stig" returns: 2`
there is one **"** after stig, is it the correct behavior for your testing?

Reply:
changed to \"$profile_ID\"